rc/definition.yaml

```diff
@@ -0,0 +1,124 @@
+---
+fields:
```

Comment:
Same goes for img. if its strictly for workflow overviews, perhaps a folder name to reflect this

Reply:
renamed as workflow-image